### DISCUSSION TITLE
寒假規劃

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1872] [General]
馬上就要迎來寒假了，不知道大家的規劃是什麼，我姑且分為以下幾種：

1、廢物型
很明顯，就是吃飽睡睡飽吃的狀態，離開床都是對床的不尊重，整天宅在家。
2、打工人型
荷包已經扁得不能再扁了，要趁假期讓它恢復正常，開學後才能繼續揮霍。
3、卷哥卷姐型
把書當成一日三餐，書就是他們的陽光、空氣、水，直接預訂下學期的卷一。
4、瘋狗型
一天有10個約，到處去玩去瘋，學期壓抑太久，再不瘋狂就等死。
5、出國型
對，就是出國玩。

這次的寒假我是1+2，很幸運接了一個家教，否則我是完全的1，每天凌晨3、4點才睡，再睡到隔天下午3、4點。不知道大家的寒假是什麼樣的呢？

**Comment by FakeWood** [comment]
我打算過個幾天的1，當個廢物休息一下，然後稍微學習一下自己有興趣的程式，也會5跟家人出國玩，也有2接家教，也想約一些很久沒見的同學敘敘舊，沒錯，1到5我全都要!

**Comment by vchien03** [comment]
過去我是100%的1(然後幻想著自己也能成為3)，但是這一兩年身體變差，有被嚇到，所以這個寒假需要調整成正常作息...不然身體真的會爛掉🥲，也約了以前的朋友們，還有安排出國，把寒假排的滿一點就比較不會讓作息那麼混亂XD

**Comment by RavenJiaSin** [comment]
我想好好享受自己的時間，開學幾乎整天都在看螢幕，寒假想去荒郊野外親近大自然，上山下海到處走走。至於讀書....不是很想呀...但好像還是得讀一下...阿...好煩。打算把暑假還沒看完的書再借回來看。

**Comment by TobywithDino** [comment]
過了煩人的學期終於可以爆睡爆玩，當然不能放過喔耶，大家都要好好享受寒假喔~ 但也別忘記充實自己白活一個月喔^^

**Comment by chuanmin0317** [comment]
感覺在這學期當中自己的能力並沒有太多，所以我希望在寒假的時候可以多學一些東西，為了都促我自己，我希望能在寒假結束以前用學到的東西作出一份成果，不過好不容易放假了，我還是會先好好的休息幾天再開始讀書吧

**Comment by lyykey** [comment]
現在是下學期第三個禮拜，我已經從寒假連續從事1到現在。
目前我的預計是延續到暑假，希望我中途不會發生期中預警，謝謝大家。

**Comment by YifenLin** [comment]
我個人是1，要在寒假起床真的太困難。尤其住在家裡又是過著茶來伸手飯來張口的日子。但我現在覺得大學生應該要好好安排自己的時間，例如增進技能，像是學吉他或是打工實習等等，越來越接近畢業，就越來越焦慮，覺得自己甚麼都不會。奉勸年輕的學弟妹們好好利用假期充實自己。

**Comment by LinChloe** [comment]
我是1、廢物型
很明顯，就是吃飽睡睡飽吃的狀態，離開床都是對床的不尊重，整天宅在家。
但為了買動漫展的周邊我還是會忍痛割愛踏出房門。
(雖然希望自己在假日能多做一點事情，但身體動不起來)

**Comment by wanka2030** [comment]
可以利用兩三個禮拜的時間去好好充實自己，或者可以走出舒適圈去挑戰其他曾未做過的挑戰。例如:可以藉由挑戰高空彈跳從高空中一躍而下，便可將上學期的壓力消逝殆盡。

**Comment by yxin214** [comment]
我是廢物型的改版？我會設立寒暑假幾個要完成的事項，但通常不會耗費很多時間，例如每天練琴半小時等，完成目標後就馬上變標準廢物型！

**Comment by yyujimmy** [comment]
我的寒假計劃屬於卷哥卷姐型，打算趁著假期好好準備下學期的考試和專題。雖然也想要好好放鬆一下，但考慮到接下來的學業壓力，覺得還是把握這段時間好好學習比較實際。不過，偶爾也會安排一些小旅行或是和朋友聚會，保持身心健康的平衡。

**Comment by KH1126** [comment]
我是1。
經過一學期的努力，還有準備期末考而爆肝。
我寒假會睡到自然醒，事情也不會塞太多。
就只需做自己喜歡的事情就好，當然除了休息還是要有一些學習。

**Comment by jeff110303549** [comment]
寒假規劃因人而異，每個人的生活方式和目標都不同。在寒假期間，有些人可能選擇放鬆休息，享受悠閒的生活；有些人可能會找工作賺取額外的收入；還有些人可能會全心投入學習，準備下個學期的考試或是知識。另外，也有人會趁著假期到處旅行，體驗不同的文化和風景。
每個人都有自己的生活方式和目標，重要的是找到適合自己的方式，讓寒假過得充實而有意義。

**Comment by 110101501** [comment]
我自己是1+5，既然都到了寒假就是要好好讓自己放鬆休息。
自己是有趁著寒假的時候出國，正好避開了寒流來的時刻
